### DISCUSSION TITLE
Deflationary mechanism to lower available $FIL in the open market.

### DISCUSSION THREAD

**Discussion by NSC-FIL** [filecoin-project/FIPs#324] [Enhancements - Misc]
Proposal to increase the "$FIL deflationary mechanism" effects to incentivize buying and holding $FIL to get the price moving up.  IMO, the continual decline in market price of $FIL hurts the adoption of our network as the value drop causes long-term holders (especially Storage Providers) to lose confidence in the economic viability of Filecoin.  Decreasing the supply of FIL in the market should help bolster the price and may create more demand for $FIL from investors, clients, and SP's.

**Comment by AxCortesCubero** [comment]
In general, I think sure it would be nice to have supply lowered so price goes up, as long as it is not a mechanism that negatively affects anything else. We already have such mechanisms: Gas fees, locking up pledge collateral, capped decaying minting rate.
So we would need a specific idea to evaluate, modifying one of these? adding another mechanism? then we can evaluate the pros and cons of that specific mechanism.

**Reply by NSC-FIL** [reply]
Please correct me if I'm wrong but:
1) The gas fees are anchored to a base fee and message fees are burned (not paid to miners aka SP's).
2) SP's must lock up FIL to seal sectors (10x collateral for verified deals) which is unlocked and returned to SP control on expiry of sectors.
3) The decayed mint rate is deflationary in that there are less FIL rewarded each winning block. I'd like to know the exact rate of deflation, can you detail the projected decay curve or a link to it? 

Taking into consideration the parameters above, the Filecoin network still produces more FIL than the demand in the open market can absorb, as the price has declined continuously since September 2021.  And without increasing the demand for FIL (by investors, clients, or storage providers), only decreasing the supply of FIL will eventually stop the market price and reverse the slumping price trend.

**Reply by NSC-FIL** [reply]
A few suggestions I have:
1) Further incentivize Storage Providers to take storage deals (rather than pledging CC sectors).  Maybe burn a matching amount of FIL equal to the collateral amount locked by the SP for a sector with deals?  This might also incentivize the SP's to use "snap deals" and lockup FIL rather than let their CC sectors expire and hence dump FIL on the market. 
2) Create a "halving event" like in Bitcoin. Periodically halve the amount of FIL rewards. (Personally as an active SP,  I would NOT prefer this, but admittedly it would decrease the inflation rate dramatically).
3) Incentivize more developers to create DeFi use cases and/or educate investors on how to use "wrapped" FIL in DeFi dapps?
4) Build in deflationary mechanisms in the upcoming FVM? Any comments or thoughts from those familiar with the Ethereum VM or FVM?
5) Create functionality (in the FVM?) to mint NFT's on Filecoin that require using FIL.  If we are going to store NFT data why not also allow minting?

Feel free to challenge or agree on any of my suggestions. ;-)

**Comment by dd45e640b42e6da7da96faee3996ef7c** [comment]
you have a point for why a decreasing FIL price is a bad thing?

**Reply by NSC-FIL** [reply]
One thing that comes to mind is that price is often taken as a sign of the health of the project, the higher the price the more successful the Filecoin ecosystem seems.  When the FIL price trends lower it gives the impression something is wrong with Filecoin.  Generally, I believe all stakeholders want the FIL price to increase in value to indicate the Filecoin network is growing (in popularity as well as market capitalization).  Sure there are other arguments to be made here.

**Reply by dd45e640b42e6da7da96faee3996ef7c** [reply]
what i see here right now: you are mixing 2 things together, one of them reasonable - one of them very dangerous.

- work towards increased utilization of FIL
vs.
- artificially remove/delay future FIL from the ecosystem to achieve an outcome that is, in my eyes, delusional

i am fully with you on FIL utilization. 100% do it, bring forward some great projects, ideas and what not - but just "limiting supply" is a very very one dimensional view of a complex situation you do not even have arguments for to be a real problem

if we raise the price by "insert crazy ecosystem change here" to raise the price so the project seems healthy - what does tell people about the project?

**Reply by dd45e640b42e6da7da96faee3996ef7c** [reply]
if you really think limiting supply would help the project to become more healthy: set an example! send your FIL to f099 and make your case to follow that example public - you might inspire others to do so. it's the fastest and easiest measure to limit the supply. be a hero.

i think you can see where my motivation to speak out against this comes from: leave my investments and their well calculated future earnings alone. thanks!

**Reply by NSC-FIL** [reply]
Actually, when my storage cluster went down for almost two weeks last year , I did send f099 more than 300 FIL in slashing penalties, enforced by the protocol.  Back then FIL was about US$70+/FIL.  Though $20,000+ seemed a heavy loss to me, that tiny bit of "burning" didn't stop FIL from sliding to it's current price.  Maybe if we could convince the top 100 SP's to voluntarily burn all their FIL rather than sell on the open market it might help turn around the FIL price.  Good luck with that!  
Sarcasm aside, I think we need to revisit the token economics and reverse the downward price trend, adjusting at the protocol level if necessary.  Many other cryptocurrency projects have done it with success.

**Reply by dd45e640b42e6da7da96faee3996ef7c** [reply]
those were locked FIL - no need to convert them to USD. You burned  300+ FIL, involuntary... 

I am just more than skeptical that limiting future supply at the current point in time would change anything regarding the price. and i am even more skeptical regarding the assumption that the current price poses a problem to the overall ecosystem.

**Reply by NSC-FIL** [reply]
Maybe more attention needs to happen in the crypto media and/or crypto social platforms to educate people on the tokenomics of $FIL. I'm throwing ideas out here because the reality is in the low price of $FIL and it's downward trajectory. Even compared to other competitors like Arweave ($AR) ... $FIL has been dropping faster and with greater trading volume. Regretfully, not a sign of strong demand.

**Comment by NSC-FIL** [comment]
For those interested here's a current list of major cryptocurrencies with deflationary mechanisms:
https://lcxwire.com/get-complete-list-of-deflationary-cryptocurrencies-2022-these-tokens-burn/
** Note Filecoin is listed.  Maybe we need to just increase the rate of burning or add another mechanism based on some other metric?

**Reply by dd45e640b42e6da7da96faee3996ef7c** [reply]
> Filecoin (FIL)
> 
> FIL is the native asset of the Filecoin blockchain, an open-source, decentralized storage, and content delivery network. The coin has a capped supply (2 billion FIL). Also, mining rewards for Filecoin are halved every six years. Plus, transaction fees paid in FIL are burned, reducing the circulating supply. Some FIL coins are also burned to fund on-chain computations and bandwidth as network message fees.

- halving - check!
- transaction fee burning - check!
- capped supply - check!
- never seen before burning mechanisms - check!

there is everything you ask for above. the problem is not the supply but the demand. limiting/decreasing future supply now to hokus-pokus-bs the price up is just....

**Reply by NSC-FIL** [reply]
Creating more demand for FIL is obviously not sufficient at this point, hence the steady and unrelenting slide in FIL price.  IMO only supply side changes can have an immediate impact to reverse the downward trajectory.  Macro market conditions change and so should the tokenomics (minimally whenever possible) to keep Filecoin moving forward.

**Reply by dd45e640b42e6da7da96faee3996ef7c** [reply]
have any proof for that? let me make an equally unthoughtful statement in the other direction: an adjustment of the FIL supply dynamics will destroy confidence in the project and lets the price slide even more.

there is still no argument that the price poses a sustained problem at all anyways

**Reply by NSC-FIL** [reply]
![supply_deamand_equilibrium](https://user-images.githubusercontent.com/70535127/158724188-9fb697b3-87e4-4591-a15b-7021cc34ca60.png)

**Reply by dd45e640b42e6da7da96faee3996ef7c** [reply]
yup. why screw with chain economics if we can influence the demand curve instead without touching the protocol?

i have the feeling that the "now price" is the point here and not the long term success of the project
